src/adagrams.js

```diff
@@ -1,15 +1,151 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letterPool = {
```

Comment:
This can be declared outside of the function so it doesn't have to be recreated each time the function is called.